waspc/cli/src/Wasp/Cli/Command/Dockerfile.hs

```diff
@@ -0,0 +1,24 @@
+module Wasp.Cli.Command.Dockerfile
```

Comment:
The thing that stinks with calling this `Dockerfile.hs` is it messes up VS Code stuff. Even if you tell it it's a Haskell file, it still seems LSP doesn't check it (for me at least). Should I change this filename? :D

Reply:
Oh wow interesting! Sure change it! DockerfileDonkey or DockerfileDodo :D

Reply:
Haha, nice naming, thanks. lol Maybe it is actually not a filename issue (well, part is, but maybe not HLS stuff)? Some of the CLI files I can get HLS to work, others not. Even if I change the name of that file and `cabal clean/build` I still get the squiggly at the top with:

```
Failed to parse result of calling cabal
Failed command: cabal --builddir=/home/shayne/.cache/hie-bios/dist-waspc-faa186440bd94206660696d24dfd040a v2-repl --with-compiler /home/shayne/.cache/hie-bios/wrapper-b54f81dea4c0e6d1626911c526bc4e36 --with-hc-pkg /home/shayne/.cache/hie-bios/ghc-pkg-9800c8020b96679951b03bc00d210a1a waspc:lib:cli-lib
Build profile: -w ghc-8.10.7 -O1

src/Wasp/Analyzer/Parser/Lexer.x:10:1: error:
    Could not load module ‘Wasp.Analyzer.Parser.LexerUtils’
    It is a member of the hidden package ‘waspc-0.5.2.1’.

src/Wasp/Analyzer/Parser/Lexer.x:11:1: error:
    Could not load module ‘Wasp.Analyzer.Parser.Monad’
    It is a member of the hidden package ‘waspc-0.5.2.1’.
```

I wonder if any recent project structure changes or the compiler merge affected this? It is weird that some old CLI files work still though (they get a squiggly after I re-open them, but I still get HLS stuff). 🤔 Do you see the same thing?

Reply:
Forgot to tag you in the question @Martinsos

Reply:
Ok, this was all related to the cache issue we fixed on Discord, resolving and leaving file named as is. 👍🏻